api.json

```diff
@@ -4489,6 +4798,219 @@
         }
       }
     },
+    "DiscountCustomerParam": {
+      "title": "Customer applicable to a discount",
+      "type": "object",
+      "properties": {
+        "customer_group_id": {
+          "description": "Customer group ID",
+          "type": "string"
+        }
+      }
+    },
+    "DiscountLineItem": {
+      "title": "Line Item, for use while applying a discount",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Line item ID.",
+          "type": "string"
+        },
+        "handle": {
+          "description": "Handle.",
+          "type": "string"
+        },
+        "product_id": {
+          "description": "Product ID",
+          "type": "string"
+        },
+        "product_brand_id": {
+          "description": "Product Brand ID.",
+          "type": "string"
+        },
+        "product_supplier_id": {
+          "description": "Product Supplier ID.",
+          "type": "string"
+        },
+        "product_type_id": {
+          "description": "Product Type ID.",
+          "type": "string"
+        },
+        "product_tag_ids": {
+          "description": "Product Tag IDs.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "variant_parent_id": {
+          "description": "Product variant parent ID",
+          "type": "string"
+        },
+        "quantity": {
+          "description": "Quantity of the line item.",
+          "type": "integer"
+        },
+        "unit_price": {
+          "description": "The unit price for the line item.",
+          "type": "number",
+          "format": "double"
+        },
+        "price_adjusted": {
+          "description": "Indicator to suggest that the price has been adjusted manually, such as through a discount.",
+          "type": "boolean"
+        },
+        "confirmed": {
+          "description": "Whether the line item is confirmed as part of a layby or on account sale.",
+          "type": "boolean"
+        },
+        "loyalty_value": {
+          "description": "The line item's loyalty value",
+          "type": "number",
+          "format": "double"
+        },
+        "explicit_loyalty_value": {
+          "description": "Whether loyalty was explicitly set in the product or it just uses the default outlet ratio.",
+          "type": "boolean"
+        },
+        "unit_discount": {
+          "description": "The line item's unit discount",
+          "type": "number",
+          "format": "double"
+        },
+        "unit_tax_discount": {
+          "description": "The line item's unit tax discount",
+          "type": "number",
+          "format": "double"
+        },
+        "promotion_ids": {
+          "description": "List of promotion IDs applied to this line item.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "metadata": {
```

Comment:
We don't actually use `metadata` field for each line item so not sure if it's worth mentioning in the doc? 🤔

Reply:
Hmm, in that case I might take it out completely.

It's either that or put a big 'UNUSED' label across it.

Reply:
I'd say better to take it out and avoid any confusion or give people something to ask about.